packages/menu/src/MenuItem.ts

```diff
@@ -92,26 +92,41 @@ export class MenuItem extends ActionButton {
 
     protected updated(changes: PropertyValues): void {
         super.updated(changes);
-        if (this.getAttribute('role') === 'option' && changes.has('selected')) {
-            this.setAttribute(
-                'aria-selected',
-                this.selected ? 'true' : 'false'
-            );
+        const role = this.getAttribute('role');
```

Comment:
Can you move this into the `if` scope for easy removal in the future?

Reply:
At least with the latest code `role` is referenced in the if and the else, so I'm gonna mark this resolved.